e2e/wdio/core/tests/dialog.e2e-spec.ts

```diff
@@ -0,0 +1,676 @@
+            clearAndCloseDialog();
+        }, 1);
+
+        it('should check resizing dialog', () => {
+            openDialog(complexDialog);
+            waitForNotDisplayed(busyIndicator);
+            const startStyle = getAttributeByName(dialogContainer, styleAttribute);
+
+            checkResizingDialog();
+            expect(getAttributeByName(dialogContainer, styleAttribute)).not.toBe(startStyle);
+            clearAndCloseDialog();
+        }, 1);
+    });
+
+    describe('stacked dialogs examples', function() {
+        it('should check that there can be multiple dialogs', () => {
+            openDialog(stackedDialog);
+
+            expect(getElementArrayLength(dialog)).toBe(1);
+            click(dialog + button, 1);
+            waitForElDisplayed(dialog, 1);
+
+            expect(getElementArrayLength(dialog)).toBe(2);
+            click(dialog + button, 3);
+            closeDialog();
+
+            expect(doesItExist(dialog)).toBe(false);
+        });
+    });
+
+    describe('custom backdrop and container examples', function() {
+        it('should check custom backdrop dialog dismissal', () => {
+            openDialog(customDialog);
+
+            click(dialog + button);
+
+            expect(doesItExist(dialog)).toBe(false);
+            openDialog(customDialog);
+
+            click(dialog + button, 1);
+
+            expect(doesItExist(dialog)).toBe(false);
+        });
+
+        it('should check custom backdrop class', () => {
+            openDialog(customDialog);
+
+            expect(getAttributeByName(dialog, classAttribute)).toContain(customClass);
+
+            click(dialog + button);
+        });
+
+        it('should check custom container dismissal', () => {
+            click(customDialog + button, 1);
+            waitForElDisplayed(dialog);
+            click(dialog + button);
+
+            expect(doesItExist(dialog)).toBe(false);
+
+            click(customDialog + button, 1);
+            waitForElDisplayed(dialog);
+            click(dialog + button, 1);
+
+            expect(doesItExist(dialog)).toBe(false);
+        });
+
+        it('should check static dialog dismissal', () => {
+            click(customDialog + button, 2);
+            waitForElDisplayed(dialog);
+            click(dialog + button);
+
+            expect(doesItExist(dialog)).toBe(false);
+
+            click(customDialog + button, 2);
+            waitForElDisplayed(dialog);
+            click(dialog + button, 1);
+
+            expect(doesItExist(dialog)).toBe(false);
+        });
+    });
+
+    describe('playground examples', function() {
+        afterEach(() => {
+            refreshPage();
+            waitForElDisplayed(dialogPage.title);
+        }, 1);
+
+        it('should check dialog hasBackdrop option', () => {
+            openDialog(playgroundDialog);
+
+            expect(executeScriptBeforeTagAttr(dialog, backgroundColor)).toContain(blackBackground);
```

Comment:
this is testing against hardcoded color. Please, either refactor and remove this check

Reply:
We have had discussions on the topic of colors, but we have made no agreements. 
For this test case, I am checking that backdrop exists. backdrop exists as a pseudo element for fd-dialog, so I see 2 ways to check it. 1. as i did above. 2. with screenshot. but screenshot for this would be very unstable because the backdrop is fullscreen and things such as the navigation bar would be in the image. So each time a new component would be added to the core repo, screenshot would fail because of changes to nav bar. For this reason, I used the above approach. 
Cannot skip/remove this check.

cc: @AntonOlkhovskyi @skocharian